picard/ui/searchdialog.py

```diff
@@ -705,21 +709,21 @@ def parse_releases(self, releases):
     def display_results(self):
         self.show_table()
         self.table.verticalHeader().setDefaultSectionSize(100)
+
         for row, release in enumerate(self.search_results):
-            table_item = QtWidgets.QTableWidgetItem
             self.table.insertRow(row)
-            self.table.setItem(row, self.colpos('name'), table_item(release.get("album", "")))
-            self.table.setItem(row, self.colpos('artist'), table_item(release.get("albumartist", "")))
-            self.table.setItem(row, self.colpos('format'), table_item(release.get("format", "")))
-            self.table.setItem(row, self.colpos('tracks'), table_item(release.get("tracks", "")))
-            self.table.setItem(row, self.colpos('date'), table_item(release.get("date", "")))
-            self.table.setItem(row, self.colpos('country'), table_item(release.get("country", "")))
-            self.table.setItem(row, self.colpos('labels'), table_item(release.get("label", "")))
-            self.table.setItem(row, self.colpos('catnums'), table_item(release.get("catalognumber", "")))
-            self.table.setItem(row, self.colpos('barcode'), table_item(release.get("barcode", "")))
-            self.table.setItem(row, self.colpos('language'), table_item(release.get("~releaselanguage", "")))
-            self.table.setItem(row, self.colpos('type'), table_item(release.get("releasetype", "")))
-            self.table.setItem(row, self.colpos('status'), table_item(release.get("releasestatus", "")))
+            self.set_table_item(row, 'name',     release, "album")
```

Comment:
This violates PEP8 https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements

Reply:
> This violates PEP8 https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements

Those spaces are meant to make things easier to read (and it does, really).

And it doesn't violate PEP8, if one reads it carefully:

> Some other good reasons to ignore a particular guideline:
> 
> When applying the guideline would make the code less readable, even for someone who is used to reading code that follows this PEP.

Reply:
Hmm, I found it weird to read, since I am not used to aligned `=` or `,` in python. If we are making it an exception to PEP8 we should add it to our contribution guide.

Reply:
That's pretty usual to "columnize" such repeated statements (using spaces, not tabs) in all languages.
We don't want to make a rule, as it is very dependent on context.

One has yet to prove that:

```python
self.set_table_item(row, 'name', release, "album")
self.set_table_item(row, 'artist', release, "albumartist")
self.set_table_item(row, 'format', release, "format")
self.set_table_item(row, 'tracks', release, "tracks")
self.set_table_item(row, 'date', release, "date")
self.set_table_item(row, 'country', release, "country")
self.set_table_item(row, 'labels', release, "label")
self.set_table_item(row, 'catnums', release, "catalognumber")
self.set_table_item(row, 'barcode', release, "barcode")
self.set_table_item(row, 'language', release, "~releaselanguage")
self.set_table_item(row, 'type', release, "releasetype")
self.set_table_item(row, 'status', release, "releasestatus")
self.set_table_item(row, 'score', release, "score", conv=int)
```
is easier to read than:

```python
self.set_table_item(row, 'name',     release, "album")
self.set_table_item(row, 'artist',   release, "albumartist")
self.set_table_item(row, 'format',   release, "format")
self.set_table_item(row, 'tracks',   release, "tracks")
self.set_table_item(row, 'date',     release, "date")
self.set_table_item(row, 'country',  release, "country")
self.set_table_item(row, 'labels',   release, "label")
self.set_table_item(row, 'catnums',  release, "catalognumber")
self.set_table_item(row, 'barcode',  release, "barcode")
self.set_table_item(row, 'language', release, "~releaselanguage")
self.set_table_item(row, 'type',     release, "releasetype")
self.set_table_item(row, 'status',   release, "releasestatus")
self.set_table_item(row, 'score',    release, "score", conv=int)
```

But you have a point, this is an intentional exception to the rule you cited, and you know i'm used to enforce PEP8.